app/views/additional_payments/claims/_eligibility_confirmed_multiple.html.erb

```diff
@@ -1,53 +1,26 @@
+      tag: "h1",
+      size: "l"
+    },
+    hint: {
+      text: "You can only make one claim per academic year. Your choice does not impact your ability to apply in future years."
+    } %>
 
-      <div id="payments-hint" class="govuk-hint govuk-!-padding-bottom-2">
-        You can only make one claim per academic year. Your choice does not impact your ability to apply in future years.
-      </div>
-
-      <%= errors_tag f.object, :selected_claim_policy %>
-
-      <div class="govuk-radios" id="claim_base" data-module="govuk-radios">
-        <% policies.each do |policy| %>
-          <div class="govuk-radios__item">
-            <%= f.radio_button(:selected_claim_policy, policy, class: "govuk-radios__input", checked: @form.selected_policy?(policy)) %>
-            <%= f.label "selected_claim_policy_#{policy.to_s.downcase}", "#{award_amount(@form.award_amount(policy))} #{policy_name(policy)}", class: "govuk-label govuk-radios__label" %>
-            <div class="govuk-hint govuk-radios__hint">
-              <%= t("#{policy.locale_key}.purpose") %>
-            </div>
-          </div>
-        <% end %>
-      </div>
-    </fieldset>
+  <%= govuk_details(summary_text: "Help with choosing an additional payment") do %>
+    For the majority of teachers, the larger payment is the most beneficial option. In rare cases, some may prefer to choose the smaller option, such as if you are close to the threshold for the higher rate of tax.
   <% end %>
 
-  <details class="govuk-details" data-module="govuk-details">
-    <summary class="govuk-details__summary">
-      <span class="govuk-details__summary-text">
-        Help with choosing an additional payment
-      </span>
-    </summary>
-    <div class="govuk-details__text">
-      For the majority of teachers, the larger payment is the most beneficial option. In rare cases, some may prefer to choose the smaller option, such as if you are close to the threshold for the higher rate of tax.
-    </div>
-  </details>
-
-  <%= f.submit "Apply now", class: "govuk-button", data: { module: "govuk-button" } %>
+  <%= f.govuk_submit "Apply now" %>
 <% end %>
 
 <p class="govuk-body">
   For more information about why you are eligible for different additional payments, read about the
-  <%= link_to "early-career payments (opens in new tab)", Policies::EarlyCareerPayments.eligibility_page_url, class: "govuk-link govuk-link--no-visited-state", target: "_blank" %>
+  <%= govuk_link_to "early-career payments (opens in new tab)", Policies::EarlyCareerPayments.eligibility_page_url, no_visited_state: true, target: "_blank" %>
   and
-  <%= link_to "levelling up premium payments (opens in new tab)", Policies::LevellingUpPremiumPayments.eligibility_page_url, class: "govuk-link govuk-link--no-visited-state", target: "_blank" %>.
+  <%= govuk_link_to "levelling up premium payments (opens in new tab)", Policies::LevellingUpPremiumPayments.eligibility_page_url, no_visited_state: true, target: "_blank" %>.
```

Comment:
Don't mean to butt in just saw this  PR in the feed - you can use the recently-added `new_tab: true` which automatically appends  `(opens in new tab)` and also sets the `rel` and `target` attrs. More info [here](https://govuk-components.netlify.app/helpers/link/#links-that-open-in-a-new-tab).

Reply:
@peteryates no not all, I've made your suggested change here. Thanks again for the great library and making our lives easier!